tests/unit/state.py

```diff
@@ -0,0 +1,11 @@
+from dvc.state import State
+from tests.basic_env import TestDvc
+
+
+class TestMtimeAndSizeReturnType(TestDvc):
+    def test(self):
+        self.create(self.DATA, self.DATA_CONTENTS)
+        time, size = State.mtime_and_size(self.DATA)
```

Comment:
Can we test the same for self.DATA_DIR too? Just so this unit test triggers the whole method.